### PR TITLE
Merges dependabot PRs into one

### PR DESCRIPTION
**Pull request by bashir2**
## Description of what I changed
Ran the [merge_all_dependabot_PRs.sh](https://github.com/google/fhir-data-pipes/blob/master/utils/merge_all_dependabot_PRs.sh) to merge all dependabot PRs into one, then reverted spring-boot version update to be Java 11 compatible.

## E2E test

TESTED:

Relied on the continuous build

## Checklist: I completed these to help reviewers :)

- [x] I have read and will follow the [review process](https://github.com/GoogleCloudPlatform/openmrs-fhir-analytics/blob/master/doc/review_process.md).
- [x] I am familiar with Google Style Guides for the language I have coded in.
- [x] My IDE is configured to follow the Google [**code styles**](https://google.github.io/styleguide/).
- [ ] I have **added tests** to cover my changes. (If you refactored existing code that was well tested you do not have to add tests)
- [x] I ran `mvn clean package` right before creating this pull request and added all formatting changes to my commit.
- [x] All new and existing **tests passed**.
- [x] My pull request is **based on the latest changes** of the master branch.

### CODE REVIEW

File: e2e-test/e2e.sh [hunk 1]
```diff
@@ -24,8 +24,19 @@ function setup() {
   docker build -t us-docker.pkg.dev/fhir-proxy-build/stable/fhir-access-proxy:${BUILD_ID} .
   docker-compose -f docker/keycloak/config-compose.yaml \
                  up --force-recreate --remove-orphans -d --quiet-pull
+  # TODO find a way to expose docker container logs in the output; currently
```
Comment (omarismail94) [inline e2e-test/e2e.sh:27]: I think we need to add a logging label in the compose files to direct logs elsewhere: 

https://docs.docker.com/config/containers/logging/configure/

for example, we can redirect to journald and print them when exiting the script:
https://stackoverflow.com/questions/43663013/access-logs-of-a-killed-docker-container

**Review by omarismail94 — APPROVED** [review]
Made a couple of comments addressing the TODOs

File: e2e-test/e2e.sh (hunk 1 above)
Comment (bashir2) [reply e2e-test/e2e.sh:27]: Yes, agreed; for now I just leave this TODO.

File: e2e-test/e2e.sh [hunk 2]
```diff
@@ -24,8 +24,19 @@ function setup() {
...
+  #   with -d we don't get any logs and it makes debugging failures difficult.
+  # TODO what does `--wait` do? It does not seem to be a documented option and
```
[the remark this answers is not in the document]
Comment (bashir2) [reply e2e-test/e2e.sh]: Hmm, I see, yeah I saw this but my version of `docker-compose` is not `docker compose` :-)
I think mine is the old Python based version and I see we are using the new one on Kokoro. I added back `--wait` and will retry e2e before submit.

**Review by bashir2 — COMMENTED** [review]
Thanks for the review.

**Pull request merged**